Additions to molecular mechanisms (intro) (#339)
greenelab/covid19-review: content/02.introduction.md

- How can we evaluate the human response to SARS-CoV-2 on a molecular level?
- What can omics technologies tell us about changes in the body following SARS-CoV-2 infection?

> SARS-CoV-2 is a relatively novel coronavirus, being third after SARS-CoV and MERS-CoV, causing severe pneumonia in humans [@doi:10.1016/bs.aivir.2018.01.001]. 

Reviewer: ```suggestion
SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century [@doi:10.1016/bs.aivir.2018.01.001]. 
```
Moved the text around a bit to not discount the 4 endemic hCoV

> Therefore, our knowledge of the molecular mechanisms by which SARS-CoV-2 infect the cells, is mostly obtained from studies on previously identified coronaviruses. 

Reviewer: ```suggestion
Therefore, most of our current knowledge of the molecular mechanisms through which SARS-CoV-2 infects cells is obtained from studies on previously identified coronaviruses. 
```

> The entry of coronaviruses into a target cell is mediated by the binding of the viral spike (S) protein to a specific cellular receptor and subsequent priming/cleavage of S protein by cellular proteases [@pmid:16139595;@pmid:32272173]. 

Reviewer: ```suggestion
The entry of a coronavirus into a target cell is mediated by the binding of the viral spike (S) protein to a specific cellular receptor and the subsequent priming/cleavage of the S protein by cellular proteases [@pmid:16139595; @pmid:32272173]. 
```

> Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease for SARS-CoV/CoV-2 entry into a target cell, respectively [@pmid:14647384; @pmid:32075877; @pmid:20926566; @pmid:32142651; @pmid:21325420]. 

Reviewer: ```suggestion
Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@pmid:14647384; @pmid:32075877; @pmid:20926566; @pmid:32142651; @pmid:21325420]. 
```

> This has steered to a leading hypothesis on the role of ACE2 and TMPRSS2 expression in the SARS-CoV-2 infected cells, tissues and organs.

Reviewer: ```suggestion
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
```
I think this is what that sentence is saying, but I'm not quite clear what you mean by the "leading hypothesis" -- if I missed the mark with this suggestion, please just let me know!

> Recent clinical investigations from COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) [@doi:10.1001/jama.2020.3786]. 

Reviewer: ```suggestion
Recent clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) [@doi:10.1001/jama.2020.3786]. 
```

> Two studies reported that SARS-CoV-2 could not be detected in the urine specimens [@doi:10.1001/jama.2020.3786; @doi:10.1001/jama.2020.3204], however, Ling et al. found four urine samples (out of 58) to be positive for SARS-CoV-2 nucleic acids [@doi:10.1097/CM9.0000000000000774]. 

Reviewer: ```suggestion
Two studies reported that SARS-CoV-2 could not be detected in the urine specimens [@doi:10.1001/jama.2020.3786; @doi:10.1001/jama.2020.3204]; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids [@doi:10.1097/CM9.0000000000000774]. 
```

> However, respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134]. 

Reviewer: ```suggestion
Respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134]. 
```

> Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidney [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@pmid:32145190], gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851] and may even cause neurological complications [@doi:10.1016/j.bbi.2020.03.031]. 

Reviewer: ```suggestion
Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidney [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@pmid:32145190], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
It can even cause neurological complications [@doi:10.1016/j.bbi.2020.03.031]. 
```

> As it is becoming clearer that SARS-CoV-2 infection can cause damage to multiple organs, we are waiting for more reliable investigations to build a consensus. 

Reviewer: ```suggestion
As it becomes clearer that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.
```
I'm not exactly sure what you were trying to communicate with this sentence, but here is one idea! I'm happy to help, just let me know how!
